braph2genesis/graphs/_MultilayerBUT.gen.m

```diff
@@ -2,9 +2,13 @@
 MultilayerBUT < MultilayerWU (g, multilayer binary undirected with fixed threaholds graph) is a multilayer binary undirected with fixed threaholds graph.
```

Comment:
typo:
MultilayerBUT < MultilayerWU (g, multilayer binary undirected with fixed **thresholds** graph) is a multilayer binary undirected with fixed **thresholds** graph.